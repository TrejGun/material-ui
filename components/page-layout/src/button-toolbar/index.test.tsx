import { render, cleanup } from "@testing-library/react";
import { ThemeProvider, createTheme } from "@mui/material";

import { ButtonToolbar } from "./index";

afterEach(cleanup);

describe("<ButtonToolbar />", () => {
  it("renders component", () => {
    const { asFragment } = render(
      <ThemeProvider theme={createTheme()}>
        <ButtonToolbar />
      </ThemeProvider>,
    );

    expect(asFragment()).toMatchSnapshot();
  });
});
