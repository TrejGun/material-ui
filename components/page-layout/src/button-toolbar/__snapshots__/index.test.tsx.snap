// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonToolbar /> renders component 1`] = `
<DocumentFragment>
  <div
    class="MuiGrid2-root MuiGrid2-container MuiGrid2-direction-xs-row css-fa87wz-MuiGrid2-root"
  />
</DocumentFragment>
`;
